### PR TITLE
[FLINT]: New version

### PR DESCRIPTION
**Pull request by thofma**
This is a prerelease of 2.8.0, so I think according to our versioning scheme this should be v"200.790.000".

@fingolfin

### CODE REVIEW

File: F/FLINT/build_tarballs.jl [hunk 1]
```diff
@@ -25,14 +25,14 @@ using BinaryBuilder, Pkg
...
-upstream_version = v"2.7.1"
+version = v"200.790.000"  # WARNING: don't change this
+upstream_version = v"2.8.0-dev"
 
 # Collection of sources required to build FLINT
 sources = [
 #    GitSource("https://github.com/wbhart/flint2.git", "12c069ea98cd8d2c1b556bbd85568c4891f126fa"),
-    ArchiveSource("https://www.flintlib.org/flint-$(upstream_version).tar.gz",
-                  "186e2fd9ab67df8a05b122fb018269b382e4babcdb17353c4be1fe364dca481e")
+    ArchiveSource("https://github.com/wbhart/flint2/archive/refs/tags/$(upstream_version).tar.gz", #"https://www.flintlib.org/flint-$(upstream_version).tar.gz",
```
Comment (thofma) [inline F/FLINT/build_tarballs.jl:34]:
```suggestion
    ArchiveSource("https://github.com/wbhart/flint2/archive/refs/tags/v$(upstream_version).tar.gz", #"https://www.flintlib.org/flint-$(upstream_version).tar.gz",
```

**Comment by giordano** [comment]
Good to go or waiting for @fingolfin's seal of approval?

**Comment by thofma** [comment]
I'd rather wait before doing something bad to our ecosystem.

**Review by fingolfin — APPROVED** [review]
Looks good to me, thanks. We should be safe anyway, as this won't be used until we update at least LoadFlint

**Pull request merged**